modelconverter/utils/docker_utils.py

```
@@ -22,12 +23,44 @@
+
+    In rootless mode the container's root user is already mapped to the
+    invoking host user, so files written by the container come out owned
+    by the host user and must NOT be chowned (doing so would map them to
+    an unmapped sub-uid, showing up as ``nobody`` on the host).
+
+    The answer cannot change within a run, so it is cached: this is a
+    round-trip to the daemon on a path that would otherwise take it once
+    per generated compose config.
+    """
+    try:
+        out = subprocess.check_output(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)